src/olympia/landfill/generators.py

```diff
@@ -40,7 +40,7 @@ def _yield_name_and_cat(num, app=None):
         yield (addon_name, cat)
 
 
-def create_addon(name, icon_type, application):
+def create_addon(name, icon_type, application, **extra_kwargs):
```

Comment:
Bonus to be able to create addons using the landfill generator function while passing extra fields, like... `id`.